### PR TITLE
[FIX] product: Allow users to edit pricelist report

### PR DESCRIPTION
**Pull request by abz-odoo**
Steps:
- Open studio
- Go to 'Reports' tab
- Select 'Pricelist' report
- Error message

opw-3207593

### CODE REVIEW

File: addons/product/report/product_pricelist_report.py [hunk 1]
```diff
@@ -17,14 +17,15 @@ def get_html(self, data):
         return self.env.ref('product.report_pricelist_page')._render(render_values)
 
     def _get_report_data(self, data, report_type='html'):
-        quantities = data['quantities'] or [1]
+        quantities = data.get('quantities', [1])
 
-        pricelist_id = data['pricelist_id'] and int(data['pricelist_id']) or None
+        data_pricelist_id = data.get('pricelist_id')
+        pricelist_id = int(data.get('pricelist_id')) if data_pricelist_id else None
```
Comment (Feyensv) [inline addons/product/report/product_pricelist_report.py:23]:
```suggestion
        pricelist_id = data_pricelist_id and int(data_pricelist_id)
```
little simplification, but not a problem if you wanna merge the current diff

**Review by Feyensv — APPROVED** [review]
Safe diff :ok_hand: , even if I'm sick of changes needed in standard code for studio :roll_eyes:

@robodoo delegate+

**Comment by abz-odoo** [comment]
robodoo r+

**Comment by abz-odoo** [comment]
robodoo retry

**Pull request closed without merging**